### DISCUSSION TITLE
image not rendering in website

### DISCUSSION THREAD

**Discussion by mohneesh7** [adityatelange/hugo-PaperMod#290] [General]
this is my code for the post, I have put basic page variables also. 
```
this hasa bee fun.

[draft]

![image](images/test.jpg)

```

The website isn't rendering the image, the image is  kept inside  `posts/images` folder.
This is the result I get.

![Capture](https://user-images.githubusercontent.com/28930038/110122805-126cc380-7de6-11eb-8dc6-fdea31dc4c78.PNG)

**Comment by kdkasad** [comment]
I think you're just putting the image in the wrong place. Read the Hugo docs and find out about how the different directories' content get transformed into the final site. For example, if you want a global images directory, it should go in `static/images/`, not `posts/images/`

Also take a look at *page bundles*: https://gohugo.io/content-management/page-bundles/#readout

**Comment by ghost** [comment]
Place your image to **`static/images`**. And also, place your `article.md` in **`content/posts`**. 

Check the value of `baseURL` in `config.yml` is it `"https://example.com"` or `"https://example.com/new-example/"`?
If you set value `baseURL` to `"https://example.com"`, then use the Markdown for example: `![image](../images/test.jpg)`. 
If you set value `baseURL` to `"https://example.com/new-example/"`, then use the Markdown for example: `![image](../../images/test.jpg)`. 

I'm made a blog with baseURL: `"https://username.github.io/blog/"`, so I use the Markdown `![image](../../images/test.jpg)`.

**Reply by foegra** [reply]
This actually worked for me:

`![image-test](/../images/image.jpg)`

**Comment by basildoesstuff** [comment]
Hi @mohneesh7 , 
Facing the same issue as well.
when I take the markdown preview in my IDE I am able to view the image but when I run `hugo server` and render out my blog I am getting the same issue.

Have you found the solution, if so could you let me know what need to be done. I tried the methods suggest above and there still seems to not have resolve the issue.
